tgui/packages/tgui/interfaces/VotePanel.tsx

```diff
@@ -242,7 +243,10 @@ const ChoicesPanel = (props) => {
                   textAlign="right"
                   buttons={
                     <Button
-                      disabled={user.singleSelection === choice.name}
+                      tooltip={user.isGhost && 'Voting is off for the dead.'}
```

Comment:
'Ghost voting was disabled by an admin for this round'

Reply:
> 'Ghost voting was disabled by an admin for this round'

Are ghosts, by their very nature, kind of dead?
And your text is 100% wrong, because it can be turned off not only for this round (considering that the original value is in the config).

Reply:
> > 'Ghost voting was disabled by an admin for this round'
> 
> Are ghosts, by their very nature, kind of dead? And your text is 100% wrong, because it can be turned off not only for this round (considering that the original value is in the config).

observers are ghosts. It was my understanding that you disable this with admin tools but if it's also in the config then drop `for this round`